1_fetch.R

```diff
@@ -0,0 +1,69 @@
+# Functions
+source("1_fetch/src/get_nwis_data.R")
+source("1_fetch/src/retry.R")
+source("1_fetch/src/bind_data.R")
+source("1_fetch/src/get_site_info_data.R")
+
+# Download NWIS and site info data and compile files
+p1_targets_list <- list(
+  tar_target(
+    site_1_data_csv,
```

Comment:
One additional step we take is to prefix each on of these target names with `p[phase num]` (so `p1_` here). This helps when you are watching the output of `tar_make()` in the console to understand how far the build is along AND when you reference targets in other phases, it makes it easier to find that target if you need to make a change or learn more about what goes into it. Adding these prefixes would require 1) editing the target name where they are created and 2) finding and fixing their target name whenever they are used by another target.

Reply:
Makes total sense. Went ahead and updated with those changes!